integration/tests/cook/test_basic.py

```diff
@@ -96,8 +96,11 @@ def test_failing_submit(self):
 
     def test_progress_update_submit(self):
         job_executor_type = util.get_job_executor_type(self.cook_url)
-        command = 'echo "progress: 25 Twenty-five percent"; sleep 1; exit 0'
-        job_uuid, resp = util.submit_job(self.cook_url, command=command, executor=job_executor_type, max_runtime=60000)
+        command = 'echo "progress: 25 Twenty-five percent in ${PROGRESS_FILE}" >> ${PROGRESS_FILE}; sleep 1; exit 0'
+        job_uuid, resp = util.submit_job(self.cook_url, command=command,
+                                         env={'EXECUTOR_PROGRESS_OUTPUT_FILE_ENV': 'PROGRESS_FILE',
+                                              'PROGRESS_FILE': 'progress.txt'},
+                                         executor=job_executor_type, max_runtime=60000)
```

Comment:
I thought we were going with the default staying as `stdout` in our test environments so these tests wouldn't need to become more complex

Reply:
Some of the other tests (e.g. `test_multiple_progress_updates_submit` and `test_multiple_rapid_progress_updates_submit`) are relying on `stdout` being the default. These tests are checking if writing progress to another file also works.